plugins/org.jkiss.dbeaver.ext.db2/OSGI-INF/l10n/bundle_zh.properties

```diff
@@ -0,0 +1,872 @@
+tree.global_metadata.node.name=\u5168\u5c40\u5143\u6570\u636e
+tree.group.node.name=\u7ec4
+tree.groupauth.node.name=\u6388\u6743
+tree.groupauths.node.name=\u6388\u6743
+tree.groups.node.name=\u7ec4
+tree.index.node.name=\u7d22\u5f15
+tree.index_columns.node.name=\u5217
+tree.indexes.node.name=\u7d22\u5f15
+tree.indexesauths.node.name=\u7d22\u5f15\u6743\u9650
+tree.information.node.name=\u6570\u636e\u5e93\u4fe1\u606f
+tree.nickname.node.name=\u522b\u540d
+tree.method.node.name=\u65b9\u6cd5
+tree.methods.node.name=\u65b9\u6cd5
+tree.methodsauths.node.name=\u65b9\u6cd5\u6743\u9650
+tree.module.node.name=\u7ec4\u4ef6
+tree.modules.node.name=\u7ec4\u4ef6
+tree.modulesauths.node.name=\u7ec4\u4ef6\u6743\u9650
+tree.mqt.node.name=MQT
+tree.mqtsauths.node.name=MQTs \u6743\u9650
+tree.mqtdep.node.name=\u4f9d\u8d56\u5173\u7cfb
+tree.mqtdeps.node.name=\u4f9d\u8d56\u5173\u7cfb
+tree.mqts.node.name=MQTs
+tree.nicknames.node.name=\u6635\u79f0
+tree.option.node.name=\u9009\u9879
+tree.options.node.name=\u9009\u9879
+tree.package.node.name=\u5305
+tree.packagedep.node.name=\u4f9d\u8d56\u5173\u7cfb
+tree.packagedeps.node.name=\u4f9d\u8d56\u5173\u7cfb
+tree.packages.node.name=\u5305
+tree.packagesauths.node.name=\u5305\u6743\u9650
+tree.parameter.node.name=\u53c2\u6570
+tree.partition.node.name=\u5206\u533a
+tree.partitions.node.name=\u5206\u533a
+tree.period.node.name=\u671f\u95f4
+tree.periods.node.name=\u671f\u95f4
+tree.procedure.node.name=\u7a0b\u5e8f
+tree.procedures.node.name=\u7a0b\u5e8f
+tree.proceduresauths.node.name=\u7a0b\u5e8f\u6388\u6743
+tree.reference_key.node.name=\u53c3\u8003
+tree.references.node.name=\u53c3\u8003
+tree.remoteserver.node.name=\u8fdc\u7a0b\u670d\u52a1\u5668
+tree.remoteservers.node.name=\u8fdc\u7a0b\u670d\u52a1\u5668
+tree.role.node.name=\u89d2\u8272
+tree.roleauth.node.name=\u6388\u6743
+tree.roleauths.node.name=\u6388\u6743
+tree.roles.node.name=\u89d2\u8272
+tree.schema.node.name=\u6a21\u5f0f
+tree.schemas.node.name=\u6a21\u5f0f
+tree.schemasauths.node.name=\u6a21\u5f0f\u6743\u9650
+tree.security.node.name=\u5b89\u5168\u6027
+tree.sequence.node.name=\u5e8f\u5217
+tree.sequences.node.name=\u5e8f\u5217
+tree.sequencesauths.node.name=\u5e8f\u5217\u6743\u9650
+tree.statements.node.name=\u751f\u547d
+tree.storage.node.name=\u5b58\u50a8
+tree.storagegroup.node.name=\u5b58\u50a8\u7fa4\u7ec4
+tree.storagegroups.node.name=\u5b58\u50a8\u7fa4\u7ec4
+tree.table.node.name=\u8868
+tree.tables.node.name=\u8868
+tree.tablesauths.node.name=\u8868\u6743\u9650
+tree.tablespace.node.name=\u8868\u683c\u7a7a\u95f4
+tree.tablespaces.node.name=\u8868\u683c\u7a7a\u95f4
+tree.tablespacesauths.node.name=\u8868\u683c\u7a7a\u95f4\u6743\u9650
+tree.triggerdep.node.name=\u4f9d\u8d56\u5173\u7cfb
+tree.triggerdeps.node.name=\u4f9d\u8d56\u5173\u7cfb
+tree.trigger.node.name=\u89e6\u53d1\u5668
+tree.triggers.node.name=\u89e6\u53d1\u5668
+tree.types.node.name=\u7c7b\u578b
+tree.udt.node.name=User Defined Type
```

Comment:
Why there are so much entries with translation to English instead of Chinese in this file? Please remove them from everywhere.

Reply:
The workload is a bit heavy. I've partially translated it. I'm doing follow-up